src/core/dataset.ts

```diff
@@ -4,9 +4,23 @@ module Plottable {
 
 export type DatasetCallback = (dataset: Dataset) => void;
 
+export class KeyFunctions {
```

Comment:
does it make more sense to store this in a module
`export module KeyFunctions {`